### DISCUSSION TITLE
posts/pinterest/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#21] [General]
# posts/pinterest/

Find your inspiration on Pinterest

https://serendiphia.net/posts/pinterest/#comments

**Comment by ghost** [comment]
Hello Edith, 

Thank you for sharing. 

I found your topic very interesting and I loved that you chose Pinterest as I think it is a great application and not everyone knows how to use it. I really liked the way that you showcase how Pinterest work. 
I find your blog very professional.

Something that I would love to see are: 

- More images about the tips you are giving, like adding images of different boards and images about where to find your saved pins for example. 
- Maybe another video explaining more in depth the most fun things about Pinterest. 

Thank you again for sharing! I love what you are doing.
